back-end/routes/SubscriptionRoutes.js

```diff
@@ -6,11 +6,13 @@ const {
   getSubscriptionById,
   updateSubscription,
   deleteSubscription,
+  getUserSubscriptions,
 } = require('../controllers/SubscriptionController');
 
 const router = express.Router();
 
 router.get('/', getAllSubscriptions);
+router.get('/user/:id', getUserSubscriptions);
 router.post('/', createSubscription);
 router.get('/:id', getSubscriptionById);
```

Comment:
Are you sure that this method working after add endpoint on 15 line?

Reply:
All this routes work correctly, I tested it